Makefile.in

```diff
@@ -1,535 +0,0 @@
-
-install-data-optlib: $(addprefix data/optlib/,$(OPTLIB))  $(DESTDIR)$(optlibdir) $(DESTDIR)$(pkgconf_optlibdir) FORCE
-	for c in $(OPTLIB); do \
-		$(INSTALL_DATA) $(srcdir)/data/optlib/$$c $(DESTDIR)$(optlibdir)/$$c && chmod 644 $(DESTDIR)$(optlibdir)/$$c; \
-	done
-install-data-preload: $(DESTDIR)$(preloaddir) $(DESTDIR)$(pkgconf_preloaddir) FORCE
-	echo > $(DESTDIR)$(preloaddir)/$(DEFAULT_PRELOAD_FILE)
-	for c in $(PRELOAD_OPTLIB); do \
-		echo "--options=$$(basename $$c .ctags)" >> $(DESTDIR)$(preloaddir)/$(DEFAULT_PRELOAD_FILE); \
-	done
-	chmod 644 $(DESTDIR)$(preloaddir)/$(DEFAULT_PRELOAD_FILE)
-
-#
-# install libexec
-#
-install-libexec: @install_libexec_targets@
-install-libexec-drivers: $(addprefix libexec/drivers/,$(DRIVERS)) $(DESTDIR)$(driversdir) FORCE
-	for x in $(DRIVERS); do \
-		$(INSTALL_PROG) $(srcdir)/libexec/drivers/$$x $(DESTDIR)$(driversdir)/$$x && chmod 755 $(DESTDIR)$(driversdir)/$$x; \
-	done
-
-#
-# rules for uninstalling
-#
-uninstall: uninstall-bin uninstall-lib uninstall-cmd uninstall-man uninstall-data uninstall-libexec
-
-uninstall-bin:
-	- rm -f $(DEST_CTAGS) $(DEST_ETAGS)
-
-uninstall-lib:
-	- rm -f $(DEST_READ_LIB) $(DEST_READ_INC)
-
-uninstall-cmd:
-	- rm -f $(DEST_READ_CMD)
-
-uninstall-man:
-	- rm -f $(DEST_CMAN) $(DEST_EMAN)
-
-uninstall-ctags:
-	- rm -f $(DEST_CTAGS) $(DEST_CMAN)
-
-uninstall-etags:
-	- rm -f $(DEST_ETAGS) $(DEST_EMAN)
-
-uninstall-data: uninstall-optlib uninstall-preload
-	- rmdir $(pkgdatadir)
-	- rmdir $(pkgsysconfdir)
-
-uninstall-optlib:
-	- rm -f $(addprefix $(optlibdir)/,$(OPTLIB))
-	- rmdir $(optlibdir)
-	- rmdir $(pkgconf_optlibdir)
-uninstall-preload:
-	- rm -f $(preloaddir)/$(DEFAULT_PRELOAD_FILE)
-	- rmdir $(preloaddir)
-	- rmdir $(pkgconf_preloaddir)
-
-uninstall-libexec: uninstall-drivers
-	- rmdir $(pkglibexecdir)
-
-uninstall-drivers:
-	- rm -f $(addprefix $(driversdir)/,$(DRIVERS))
-	- rmdir $(driversdir)
-
-#
-# miscellaneous rules
-#
-tags: $(CTAGS_EXEC)
-	./$(CTAGS_EXEC) $(srcdir)/*
-
-TAGS: $(CTAGS_EXEC)
-	./$(CTAGS_EXEC) -e $(srcdir)/*
-
-clean: clean-units clean-gcov clean-tmain
-	$(SILENT) echo Cleaning top level
-	$(SILENT) rm -f $(OBJECTS) $(CTAGS_EXEC) tags TAGS $(READ_LIB)
-	$(SILENT) rm -f $(SOURCES:.c=.gcno)
-
-mostlyclean: clean
-
-distclean: clean
-	rm -f $(CONFIG_GEN)
-
-maintainerclean: distclean
-	rm -f $(AUTO_GEN)
-	rm -rf autom4te.cache/
-
-#
-# implicit rules
-#
-.c.$(OBJEXT):
-	$(SILENT) dir="$$(dirname $@)"; mkdir -p "$$dir"
-	$(V_CC) $(CC) -I. -I$(srcdir) -I$(srcdir)/main $(DEFS) $(ALL_CPPFLAGS) $(ALL_CFLAGS) -o $@ -c $<
-$(addsuffix .$(OBJEXT),$(basename $(notdir $(REGEX_SOURCES)))): ALL_CPPFLAGS += \
-	$(EXTRA_CPPFLAGS)
-
-REV := $(shell git rev-parse --short HEAD 2>/dev/null || echo 0)
-ALL_CPPFLAGS += -DCTAGS_COMMIT_ID=0x$(REV)
-
-options.$(OBJEXT): commit-stamp
-entry.$(OBJEXT): commit-stamp
-
-commit-stamp: FORCE
-	echo "$(REV)" > $@.tmp
-	if diff -q $@ $@.tmp; then rm -f $@.tmp; else mv -f $@.tmp $@; fi
```

Comment:
Is `commit-stamp` target not available in `Makefile.am`?
I don't see the commit-id in output of `ctags --version`.